spinalcordtoolbox/scripts/sct_fmri_moco.py

```diff
@@ -172,8 +180,47 @@ def main(argv: Sequence[str]):
     if not (is_qc_none == is_seg_none):
         parser.error("Both '-qc' and '-qc-seg' are required in order to generate a QC report.")
 
-    # run moco
-    fname_output_image = moco_wrapper(param)
+    # Run moco
+    if arguments.dl:
+        if moco_dl is None:
+            raise ImportError("mocoDL module not found. Please ensure SCT was installed with mocoDL support.")
```

Comment:
I don't think we need this check? Since, if the `moco_dl` module doesn't exist, then the `ImportError` will be thrown automatically by Python at the top of the file.